tests/test_e2e_40_sdntrace.py

```diff
@@ -772,3 +772,172 @@ def test_060_run_sdntrace_order(cls):
+                            }
+                        }
+                    },
+                    {
+                        "trace": {
+                            "switch": {
+                                "dpid": "00:00:00:00:00:00:00:01",
+                                "in_port": 2
+                            }
+                        }
+                    },
+                    {
+                        "trace": {
+                            "switch": {
+                                "dpid": "00:00:00:00:00:00:00:01",
+                                "in_port": 2
+                            },
+                            "eth": {
+                                "dl_vlan": 10
+                            }
+                        }
+                    }
+        ]
+
+        api_url = KYTOS_API + '/amlight/sdntrace_cp/traces'
+        response = requests.put(api_url, json=payload)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        list_results = data["result"]
+
+        assert list_results[0][0]["type"] == "last"
+        assert list_results[0][0]["out"]["port"] == 1
+
+        assert list_results[1][0]["type"] == "last"
+        assert list_results[1][0]["out"]["port"] == 1
+
+        assert len(list_results[2]) > 1
+        assert len(list_results[3]) > 1
+
+    def test_075_run_sdntrace_any_vlan(cls):
+        """Run SDNTrace to test run_sdntrace when vlan is any"""
+        payload_stored_flow = {
+            "flows": [
+                {
+                    "priority": 100,
+                    "match": {
+                        "in_port": 2,
+                        "dl_vlan": 100
+                    },
+                    "actions": [
+                        {
+                            "action_type": "output",
+                            "port": 1
+                        }
+                    ]
+                },
+                {
+                    "priority": 10,
+                    "match": {
+                        "in_port": 2,
+                        "dl_vlan": 10
+                    },
+                    "actions": [
+                        {
+                            "action_type": "output",
+                            "port": 2
+                        }
+                    ]
+                }
+            ]
+        }
+        api_url = KYTOS_API + '/kytos/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
+        response = requests.post(api_url, json = payload_stored_flow)
+        assert response.status_code == 202, response.text
+        time.sleep(10)
+
+        payload = [
+                    {
+                        "trace": {
+                            "switch": {
+                                "dpid": "00:00:00:00:00:00:00:01",
+                                "in_port": 2
+                            },
+                            "eth": {
+                                "dl_vlan": "any"
```

Comment:
Same as my prior comment regarding "untagged" in the payload, we shouldn't have "any" in the request payload.